examples/code/imgui_example/imgui_example.cpp

```diff
@@ -109,21 +109,21 @@ namespace
         style.Colors[ImGuiCol_HeaderHovered] = foreground_dark_highlight;
         style.Colors[ImGuiCol_HeaderActive] = foreground;
 
-        style.Colors[ImGuiCol_Column] = foreground_dark;
-        style.Colors[ImGuiCol_ColumnHovered] = foreground_light;
-        style.Colors[ImGuiCol_ColumnActive] = foreground;
+        // style.Colors[ImGuiCol_Column] = foreground_dark;
```

Comment:
same as a previous comment, can we remove the commented code or figure out the correct enums to set?

Reply:
Correlating to the changelog, replaced with correct enums.